site/public/css/server.css

```diff
@@ -1514,45 +1514,10 @@ end of styles used in the admin gradeable page
     flex-direction: row;
 }
 
-.wrapper-example .wrapper-top,
-.wrapper-example .wrapper-left,
-.wrapper-example .wrapper-body,
-.wrapper-example .wrapper-right,
-.wrapper-example .wrapper-bottom {
-    flex-grow: 1;
-    border: 1px solid var(--text-black);
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    text-align: center;
-}
-
 .wrapper-example .wrapper-body {
```

Comment:
You can still remove wrapper-example, wrapper-body, and wrapper-center as they are only declared classes in this CSS file and not used elsewhere.